content/docs/web-components.md

```diff
@@ -22,12 +22,12 @@ class HelloMessage extends React.Component {
 
 > Nota:
 >
-> Los `Web Component` a menudo exponen una API imperativa. Por ejemplo, un `Web Component` `video` podría exponer las funciones `play()` y `pause()`. Para acceder a un API imperativa de un `Web Component`, necesitarás utilizar una referencia para interactuar con el DOM directamente. Si estás utilizando `Web Component` de terceros, lo mejor sería escribir un componente React que sirva como un contenedor para tu `Web Component`.
+> Los `Web Components` a menudo exponen una API imperativa. Por ejemplo, un `Web Components` `video` podría exponer las funciones `play()` y `pause()`. Para acceder a un API imperativa de un `Web Components`, necesitarás utilizar una referencia para interactuar con el DOM directamente. Si estás utilizando `Web Components` de terceros, lo mejor sería escribir un componente React que sirva como un contenedor para tu `Web Components`.
```

Comment:
In this cases un Web Component is singular:
* Por ejemplo, un `Web Component` `video` podría exponer...
* Para acceder a un API imperativa de un `Web Component`... 
but in this one is OK to have it plural:
* Si estás utilizando `Web Components` de terceros
:-)